static/js/containers/ProfilePage.js

```diff
@@ -147,7 +148,11 @@ class ProfilePage extends React.Component {
       })
     ));
 
+    let text = `Welcome ${profile.preferred_name || SETTINGS.name}, let's
+    complete your enrollment to MIT Micromasters.`;
+
     return <div className="card">
+      <Jumbotron profile={profile} text={text} />
       <div className="card-copy">
```

Comment:
Ah, I missed this, I think I need to delete that text anyway (based on the mockup)